x-pack/legacy/plugins/apm/server/lib/apm_telemetry/index.ts

```diff
@@ -4,9 +4,57 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export {
-  storeApmTelemetry,
-  createApmTelementry,
-  APM_TELEMETRY_DOC_ID
-} from './apm_telemetry';
-export { makeApmUsageCollector } from './make_apm_usage_collector';
+import { Server } from 'hapi';
+import { countBy } from 'lodash';
+import { SavedObjectAttributes } from 'src/core/server';
+import { InternalCoreSetup } from 'src/core/server';
+import { isAgentName } from '../../../common/agent_name';
+import { getSavedObjectsClient } from '../helpers/saved_objects_client';
+export const APM_TELEMETRY_DOC_ID = 'apm-telemetry';
```

Comment:
Is it possible to rename this to `apm-services-telemetry` to indicate it is not all apm telemetry data?

Reply:
Yes we can do this, but it might be better to do this in the next minor version since we're so close to release, and I don't want to alarm folks by triggering a re-mapping of apm saved-objects.

Reply:
>  since we're so close to release, and I don't want to alarm folks by triggering a re-mapping of apm saved-objects

I think I said 7.5 as well at some point but APM UI Indices is in 7.6, not 7.5, right? So this should go into 7.x not 7.5.

Reply:
This PR is targeted for 7.5, so I'm hesitant to make that change here, but sure i can do it in a another PR targeted for 7.x.